website/docs/r/mks_nodegroup_v1.html.markdown

```diff
@@ -25,6 +25,7 @@ resource "selectel_mks_nodegroup_v1" "nodegroup_1" {
   volume_type       = "fast.ru-7a"
 
   install_nvidia_device_plugin = false
+  preemptible       = false
```

Comment:
we need to align equal signs in accordance with the [Terraform rules](https://developer.hashicorp.com/terraform/language/style#code-formatting)